spring-auto-restdocs-core/src/main/java/capital/scalable/restdocs/javadoc/JavadocReaderImpl.java

```diff
@@ -31,86 +33,121 @@
 
 public class JavadocReaderImpl implements JavadocReader {
     private static final Logger log = getLogger(JavadocReader.class);
+    private static final String PATH_DELIMITER = ";";
```

Comment:
for every array-like configuration property, spring always uses `comma-separated` list. check it out on https://docs.spring.io/spring-boot/docs/current-SNAPSHOT/reference/htmlsingle/